completions/quick-lint-js.bash

```diff
@@ -14,20 +18,24 @@ _quick-lint-js () {
                         return
                         ;;
                 --output-format=*)
-                        COMPREPLY=($(compgen -W 'gnu-like vim-qflist-json emacs-lisp' -- "${cur#*=}"))
+                        COMPREPLY=($(compgen -W "${_quick_lint_js_output_formats}" -- "${cur#*=}"))
                         return
                         ;;
                 --diagnostic-hyperlinks=*)
-                        COMPREPLY=($(compgen -W 'auto always never' -- "${cur#*=}"))
+                        COMPREPLY=($(compgen -W "${_quick_lint_js_diagnostic_hyperlink_modes}" -- "${cur#*=}"))
+                        return
+                        ;;
+                --exit-fail-on=*|--vim-file-bufnr=*|--path-for-config-search=*)
```

Comment:
`--path-for-config-search` accepts a (possibly-fake) file path, but pressing tab after typing `--path-for-config-search=` offers no completions.

This was broken before your patch too, but `--path-for-config-search` wasn't mentioned before your patch.